hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/GetPartitionsMessageTask.java

```diff
@@ -61,7 +62,7 @@ protected ClientMessage call() {
             Address owner = partitions[i].getOwnerOrNull();
             int index = -1;
             if (owner == null) {
-                return null;
+                return VoidResultParameters.encode();
```

Comment:
I am in favor of returning always same type of message for simplicity. I am using VoidResultParameters when client side not expecting to read what is returned. 

To return null, I added a boolean to EntryView to  differentiate if it is null or not.
But for this case, it  does not seem a good fit to me. How about we throw an exception when null ?
Or returning an empty list for addresses and indices ? The later seems like a better choice to me. 
What do you think?